packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -146,13 +146,14 @@ export const PageBlock: FunctionComponent<PageBlockProps> = ({
                   zIndex: 1,
                 }}
               >
-                {pageComments?.map((comment) => (
-                  <CommentThread
-                    key={comment.entityId}
-                    pageId={entityId}
-                    comment={comment}
-                  />
-                ))}
+                {!readonlyMode &&
```

Comment:
I think this should be above:

```
{readonlyMode || loadingTypeSystem ? null : (
          <PageSectionContainer 
...
```

Reply:
Done in https://github.com/hashintel/hash/pull/1340/commits/8bb13cef78aee51859dac9347881d8f4f79268dd